en/orm/entities.rst

```diff
@@ -461,7 +461,7 @@ converting the associated data into the correct format.
 Exposing Virtual Properties
 ---------------------------
 
-By default virtual properties are not exported when converting entities to
+By default virtual fields are not exported when converting entities to
```

Comment:
I am on the opposite fence. When seeing the PR I was explicitly looking for if the string `irtual field` still exists because that is what many people are looking for. The headers are consistent. That's important.